libmozdata/patchanalysis.py

```diff
@@ -563,6 +563,21 @@ def attachmenthandler(attachments, bugid):
         if info['uplift_comment'] and info['uplift_author']:
             break
 
+    # Landing dates per useful channels
+    all_channels = ['nightly', 'aurora', 'beta', 'release']
+    if channel not in all_channels:
+        raise Exception('Invalid channel {}'.format(channel))
+    channels = all_channels[:all_channels.index(channel) + 1]
+    landing_patterns = Bugzilla.get_landing_patterns(channels=channels)
+    landing_comments = Bugzilla.get_landing_comments(bug['comments'], [], landing_patterns)
```

Comment:
No need to call `Bugzilla.get_landing_patterns`. You should be able to simply do `Bugzilla.get_landing_comments(bug['comments'], channels)`.